packages/playground/package.json

```diff
@@ -10,7 +10,7 @@
     "dev:razzle": "razzle start",
     "start": "concurrently -c \"blue,cyan\" \"pnpm:dev:tsc\" -k \"pnpm:dev:razzle\"",
     "start:prod": "NODE_ENV=production node build/server.js",
-    "build": "tsc -b && razzle build",
+    "build": "tsc -b && razzle build --noninteractive",
```

Comment:
This should be submit in a separate PR, also not sure if we need this. I'm ok with both.

Reply:
I think we do need this, as sometimes the CLI prompts for yes/no. And if we don't disable this, the CI will fail, which was exactly the case for me last night.

![image](https://user-images.githubusercontent.com/12833674/151284515-79675327-6dc6-40d5-bb5f-a09afda836da.png)

Reply:
Interesting, why CI passed before?

Reply:
No idea. I would also like to know why. 😂
Anyways, I will try to remove this change from this PR, and open up a new one.

Reply:
Interesting enough, I reverted the changes made to disable the interactive build, but this time the CI is working...
Not sure what went wrong earlier, but now it seems the changes will no longer be needed, at least for now

Reply:
@gao-sun you can resolve this comment now. The code change has been reverted.